ruby-lsp.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency("language_server-protocol", "~> 3.17.0")
-  s.add_dependency("prism", ">= 0.19.0", "< 0.22")
+  s.add_dependency("prism", ">= 0.22.0", "< 0.25")
```

Comment:
I think it's better to make it loose so people wouldn't be blocked from upgrading ruby-lsp due to other libraries using slightly older prism. It may not be the case this time but that's how I'd approach this in general.